content/blog/dogcow/2020-08-01_july_2020_activity_report.md

```diff
@@ -0,0 +1,103 @@
++++
+type = "blog"
+title = "July 2020 Activity Report"
+author = "dogcow"
+date = "2020-08-01 15:53:09-06:00"
+tags = []
++++
+
+Welcome to the July, 2020 Activity Report for the Haiku project!
+
+This report covers hrev54370 through hrev54484.
+
+## Applications
+
+Humdinger fixed bug #16321 in the ShowImage application. The timer now responds properly to user input.
+
+Andrew Lindesay added support for HaikuDepot Server repo meta-data to be artificially matched against multiple repos. Improvements were also made in HaikuDepot's logging mechanism to de-clutter the code and enable better systematic logging.
+
+pulkomandy fixed bug #8702, allowing the system to better distinguish video Ogg files from audio.
+
+## app_server
+
+Extra Bview drawing modes required for WebKit canvas support were implemented by Kacper Kasper.
```

Comment:
KapiX (I don't really care all that much though)

Reply:
Fixed, thanks!